tests/data/test1299

```diff
@@ -36,6 +36,11 @@ Send "OPTIONS *" with --request-target
 <command>
 --request-target '*' -X OPTIONS http://%HOSTIP:%HTTPPORT/ -H "Testno: %TESTNUMBER"
 </command>
+<setenv>
+# Needed for MSYS2 to not treat the '*' as a POSIX path list
+# that has to be converted to Windows paths
+MSYS2_ARG_CONV_EXCL=*
```

Comment:
Are there any cases where this functionality is desired for tests? Maybe we could just disable this once and for all in runtests.pl?

Reply:
Yes, I think it could be global. But, in this case the trick didn't work, and I wonder what controls the expansion. (assuming it's indeed glob-related.)

I also wonder what is the catch-all setting that disables all MSYS2 and mingw-runtime command-line weirdness.

There was also this in earlier versions of `appveyor.yml`:
```
MSYS2_ARG_CONV_EXCL: '/*'
```
https://www.msys2.org/wiki/Porting/